mix.exs

```diff
@@ -23,7 +23,7 @@ defmodule Memento.Mixfile do
       # Project
       app:           @app,
       version:       @version,
-      elixir:        "~> 1.4",
+      elixir:        "~> 1.7",
```

Comment:
Bump for `:ex_unit_clustered_case`?

Since `:ex_unit_clustered_case` is test only we could just leave as-is and only run Travis against 1.7+. But, I would think the long tail of users falls within 1.7+ these days.

Reply:
Yes, makes sense. The travis config needs to be updated anyways.